app/src/test/java/com/malibin/study/github/data/repository/DefaultGithubProfileRepositoryTest.kt

```diff
@@ -0,0 +1,123 @@
+package com.malibin.study.github.data.repository
+
+import com.google.common.truth.Truth.assertThat
+import com.malibin.study.github.data.source.GithubProfileSource
+import com.malibin.study.github.domain.profile.GithubProfile
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.confirmVerified
+import io.mockk.mockk
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertAll
+
+internal class DefaultGithubProfileRepositoryTest {
+    private lateinit var fakeLocalGithubProfileSource: GithubProfileSource
+    private lateinit var fakeRemoteGithubProfileSource: GithubProfileSource
+    private lateinit var defaultGithubProfileRepository: DefaultGithubProfileRepository
+    lateinit var gitHubProfile: GithubProfile
+
+    @BeforeEach
+    fun `setUp`() {
+        fakeRemoteGithubProfileSource = mockk()
+        fakeLocalGithubProfileSource = mockk()
+        defaultGithubProfileRepository = DefaultGithubProfileRepository(
+            fakeLocalGithubProfileSource, fakeRemoteGithubProfileSource
+        )
+        gitHubProfile = GithubProfile(
+            id = 1L,
+            userName = "name1",
+            avatarUrl = "https://www.linkpicture.com/view.php?img=LPic63df5f35265bb1785508137",
+            name = "name1",
+            bio = "Hello",
+            followersCount = 1,
+            followingCount = 1
+        )
+    }
+
+    @Test
+    fun `GithubProfile이 Local DB에 없으면, Remote의 getGithubProfile()과 Local의 saveGithubProfile이 호출된다`() =
+        runBlocking {
+            //given
+            coEvery { fakeLocalGithubProfileSource.getGithubProfile("name1") } returns Result.failure(
+                IllegalArgumentException("cannot find githubProfile of userName(name1)")
+            )
+            coEvery { fakeRemoteGithubProfileSource.getGithubProfile("name1") } returns Result.success(
+                gitHubProfile
+            )
```

Comment:
저는 이렇게 정렬하는 게 더 잘 읽히더라구요 😊
```suggestion
            coEvery { 
                fakeRemoteGithubProfileSource.getGithubProfile("name1")
            } returns Result.success(gitHubProfile)
```